app/javascript/styles/components/Spotlights.scss

```diff
@@ -159,6 +159,10 @@
   display: -ms-grid;
   -ms-grid-columns: 1fr 1fr 1fr 1fr 1fr;
 
+  &__title {
+    margin-left: -20px;
```

Comment:
Would this work with rem units as well?

Reply:
Probably, but I went with pixels because it's mirroring how the `lift` class (styling the category grid) is already written with 20 pixel margins. If we rewrite it in rem units, I think we should rewrite both--what do you think?

Reply:
let's leave it for now